update/images.go

```diff
@@ -38,24 +38,30 @@ func (r ImageRepos) GetRepoImages(repo image.Name) ImageInfos {
 // ImageInfos is a list of image.Info which can be filtered.
 type ImageInfos []image.Info
```

Comment:
No-look suggestion: you could have two aliases here, to enforce where a list of images must be sorted in order to proceed. Might be more trouble than it's worth, though.

Reply:
good suggestion. made an attempt!